app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -29,7 +29,11 @@ snippet: true
 
   # tell Anaconda what to pass off to kickstart server
   # both current and legacy syntax provided
-  options.push("kssendmac", "ks.sendmac", "inst.ks.sendmac")
+  if (is_fedora && os_major >= 33) || (rhel_compatible && os_major >= 9)
+    options.push("inst.ks=#{foreman_url('provision')}", "inst.ks.sendmac")
+  else
+    options.push("ks=#{foreman_url('provision')}", "kssendmac", "ks.sendmac", "inst.ks.sendmac")  
```

Comment:
Nitpick: When rebasing I think you can drop `inst.ks.sendmac` from the else case, we were sending it "just for case" but obviously the ks.sendmac should work up until F33.